content/06.discussion.md

```diff
@@ -103,7 +103,7 @@ Despite the success and popularity of deep learning, some deep learning models c
 Researchers are actively working on modifications to deep learning frameworks to enable them to handle probability and embrace uncertainty.
 Most notably, Bayesian modeling and deep learning are being integrated with renewed enthusiasm.
 As a result, several opportunities for innovation arise: understanding the causes of model uncertainty can lead to novel optimization and regularization techniques, assessing the utility of uncertainty estimation techniques on various model architectures and structures can be very useful to practitioners, and extending Bayesian deep learning to unsupervised settings can be a significant breakthrough [@tag:gal_thesis].
-Unfortunately, uncertainty quantification techniques are underutilized in the computational biology communities and largely ignored in the current deep learning for biomedicine literature.
+Unfortunately, uncertainty quantification techniques are underutilized in the computational biology communities and largely ignored in the current deep learning for biomedical literature.
```

Comment:
```suggestion
Unfortunately, uncertainty quantification techniques are underutilized in the computational biology communities and largely ignored in the current deep learning for biomedicine literature.
```
I think the original phrasing was intended.  The parsing is "(deep learning for biomedicine) literature" not "deep learning for (biomedicine literature)".

Reply:
You're right, reading error on my part.